component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/constant/Constants.java

```diff
@@ -50,6 +50,21 @@ public class Constants {
     public static final String EMAIL_OTP_RESEND_THROTTLE_INTERVAL = "emailOtp.resendThrottleInterval";
     public static final String EMAIL_OTP_SHOW_FAILURE_REASON = "emailOtp.showValidationFailureReason";
     public static final String EMAIL_OTP_MULTIPLE_SESSIONS_ENABLED = "emailOtp.isEnableMultipleSessions";
+    public static final String EMAIL_OTP_LOCK_ACCOUNT_ON_FAILED_ATTEMPTS = "emailOtp.lockAccountOnFailedAttempts";
+
+    public static final String PROPERTY_LOGIN_FAIL_TIMEOUT_RATIO = "account.lock.handler.login.fail.timeout.ratio";
+    public static final String PROPERTY_ACCOUNT_LOCK_ON_FAILURE = "account.lock.handler.enable";
+    public static final String PROPERTY_ACCOUNT_LOCK_ON_FAILURE_MAX = "account.lock.handler.On.Failure.Max.Attempts";
```

Comment:
shall we make this consistent with the other properties?

Reply:
this is an already defined property. will improve the code to read from the existing constant